CodeGen/README.md

```diff
@@ -18,6 +18,39 @@ The workflow falls into the following architecture:
 
 ![architecture](./assets/img/codegen_architecture.png)
 
+The mega flow of the CodeGen application, from user's input query to the application's output response, is as follows:
```

Comment:
Since no microservice and megaservice mentioned here, users will confuse about the flow.
Could you put this under "Start MicroServices and MegaService" for both Gaudi and Xeon README?